quarkchain/evm/messages.py

```diff
@@ -137,6 +137,12 @@ def validate_transaction(state, tx):
     if not tx.sender:  # sender is set and validated on Transaction initialization
         raise UnsignedTransaction(tx)
 
+    if (
+        state.qkc_config.ENABLE_EIP155_SIGNER_TIMESTAMP is not None
+        and state.timestamp < state.qkc_config.ENABLE_EIP155_SIGNER_TIMESTAMP
+    ):
+        check(tx.version != 2, "EIP155 Signer has not enable yet.")
```

Comment:
check() is used for internal check.  Here we should raise an error here.

Reply:
Further, should the condition be tx.version <= 1?

Reply:
updated. 
for "Further, should the condition be tx.version <= 1?", I think this only needs to control version 2 tx, and if other versions added, we need to update the code then.